apps/frontend/src/pages/quiz/quiz.tsx

```diff
@@ -14,9 +21,16 @@ import {
 import { STEP_INCREMENT } from "./libs/constants/constants.js";
 import { Step } from "./libs/enums/enums.js";
 
+const ZERO = 0;
```

Comment:
The same with emun

Reply:
I think we have ZERO_INDEX variable in shared

Reply:
We also should have NumericValue enum in shared